setup.py

```diff
@@ -40,7 +40,7 @@
         'encryption': [
             'pynacl==0.3.0'],
         'cwl': [
-            'cwltool==1.0.20160413143011']},
+            'cwltool==1.0.20160425140546']},
```

Comment:
@tetron would it be possible to have here a `>=` better than `==`? As I did in #827 , otherwise we will run into the same problem again. What do you think?